web/src/main/webapp/WEB-INF/config/config-service-search.xml

```diff
@@ -41,7 +41,11 @@
 
         // TODO : improve
         ]]></documentation>
-      <class name=".services.main.XmlSearch"/>
+      <class name=".services.main.XmlSearch">
```

Comment:
Move this parameter to the settings with a proper explanation of the backwards compatibility problems, and I would say we can approve this.